lighthouse-cli/cli-flags.js

```diff
@@ -11,14 +11,25 @@ const yargs = require('yargs');
 const pkg = require('../package.json');
 const printer = require('./printer');
 
+/**
+ * Remove in Node 11 - [].flatMap
+ * @param {Array<Array<string>>} arr
+ * @return {string[]}
+ */
+function flatten(arr) {
+  /** @type {string[]} */
+  const result = [];
+  return result.concat(...arr);
+}
+
 /**
  * @param {string=} manualArgv
  * @return {LH.CliFlags}
  */
 function getFlags(manualArgv) {
   // @ts-ignore yargs() is incorrectly typed as not returning itself
   const y = manualArgv ? yargs(manualArgv) : yargs;
-  return y.help('help')
+  const argv = y.help('help')
```

Comment:
maybe add a comment about ``// Intentionally left as type `any` because @types/yargs doesn't chain correctly.`` or something just so it doesn't look like a glaring error later